dali/CMakeLists.txt

```diff
@@ -143,8 +143,20 @@ if (BUILD_PYTHON)
 
   add_custom_command(
     TARGET check-python
-    WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/qa/TL0_python-self-test/
-    COMMAND PYTHONPATH=${PYTHONPATH} ${PROJECT_SOURCE_DIR}/qa/TL0_python-self-test/test_no_setup.sh
+    WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/qa/TL0_python-self-test-core/
+    COMMAND PYTHONPATH=${PYTHONPATH} ${PROJECT_SOURCE_DIR}/qa/TL0_python-self-test-core/test_no_setup.sh
+  )
+
+  add_custom_command(
+    TARGET check-python
+    WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/qa/TL0_python-self-test-readers-decoders/
+    COMMAND PYTHONPATH=${PYTHONPATH} ${PROJECT_SOURCE_DIR}/qa/TL0_python-self-test-readers-decoders/test_no_setup.sh
+  )
+
+  add_custom_command(
+    TARGET check-python
+    WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/qa/TL0_python-self-test-operators/
+    COMMAND PYTHONPATH=${PYTHONPATH} ${PROJECT_SOURCE_DIR}/qa/TL0_python-self-test-operators/test_no_setup.sh
```

Comment:
Maybe you could use `qa/TL0_python-self-test/test.sh`? Since it's a gather-all test?

Reply:
No, cause it involves running the pip installs, and here we run the scripts without using pip, and assuming that you already have the deps.